engine/src/srvcgi.cpp

```diff
@@ -716,10 +716,11 @@ static void cgi_fix_path_variables()
     // SN-2014-07-29: [[ Bug 12865 ]] When a LiveCode CGI script has a .cgi extension and has
     //  the appropriate shebang pointing to the LiveCode server, PATH_TRANSLATED is not set by Apache.
     //  The current file (stored in SCRIPT_FILENAME) is the one containing the script.
+	// SN-2015-05-14: [[ MCStringGetCString Removal ]] Use ConvertToCString
 	if (MCS_getenv(MCSTR("PATH_TRANSLATED"), env))
-		t_path = strdup(MCStringGetCString(env));
+		/* UNCHECKED */ MCStringConvertToCString(env, t_path);
```

Comment:
We need to modify t_path, so an autocstring lock isn't suitable here - but use an autopointer for t_path.